### DISCUSSION TITLE
QSO date range for map + main dashboard

### DISCUSSION THREAD

**Discussion by ofadam** [magicbug/Cloudlog#758] [General]
From what I can tell, the map on the initial dashboard is limited to a week-long period. Would it be worth matching the QSO list below that with a week's worth of contacts as well?

I'll admit that I'm not someone who is terribly successful at contesting, so this might just be too resource intensive due to the potential list size.

**Comment by M0TZO** [comment]
The map on the dashboard is 1 month (can't recall if 30/31 days) but it's more than a week.

**Reply by M0TZO** [reply]
Why did I think was a month? ¯\_(ツ)_/¯

**Comment by magicbug** [comment]
It's 7 days and will stay as is out the box due to the fact loading a month of QSOs for some could be 400-1000 which is a lot of pins to load

**Comment by magicbug** [comment]
If you want a custom map for any reason in dev is map/custom it's in the code base just not in the frontend menu that allows you to do maps between two dates
